external_search.py

```diff
@@ -479,6 +483,18 @@ def remove_work(self, work):
         if self.exists(**args):
             self.delete(**args)
 
+    def _run_self_tests(self, _db, search_service):
+        def _search_for_term():
+            [search_term] = [x.value for x in search_service.settings if x.key == "test_search_term"]
```

Comment:
You should be able to do `search_service.setting("test_search_term").value_or_default("test")` here.